dockers/docker-database/supervisord.conf

```diff
@@ -3,10 +3,19 @@ logfile_maxbytes=1MB
 logfile_backups=2
 nodaemon=true
 
-[program:redis-server]
-command=/usr/bin/redis-server /etc/redis/redis.conf
+[program:start.sh]
+command=/usr/bin/start.sh
```

Comment:
It's just because we are following the similar design - start.sh controls all - in other dockers. As redis-server in docker-database does not have any dependency, I can decouple it with start.sh. 

---
In reply to: [130537997](https://github.com/Azure/sonic-buildimage/pull/808#discussion_r130537997) [](ancestors = 130537997)